configuration.packages.json

```diff
@@ -157,7 +157,7 @@
     },
     "EdFi.Ods.Common": {
       "PackageName": "EdFi.Suite3.Ods.Common",
-      "PackageVersion": "7.0.32",
+      "PackageVersion": "7.0.38",
```

Comment:
@simpat-adam I think this package change should not be included. EdFi.Suite3.Common package changed, but not  EdFi.Suite3.Ods.Common package